charts/thanos/CHANGELOG.md

```diff
@@ -14,9 +14,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Updated
 - Changed
 - Fixed
+  - issue in Rule config map.
```

Comment:
Could you remove this line?

Reply:
fixed in https://github.com/stevehipwell/helm-charts/pull/641/commits/08424dcc0f18956247d926f16a94b5f55099f943